src/components/TweetCard/TweetCard.tsx

```diff
@@ -23,15 +23,18 @@ export const TweetCard: FunctionComponent<ITweetCardProps> = ({
 
   const message = useMemo(() => {
     const text = tweet.text
-      .replace(
-        /(?:((?<=[\s\W])|^)[#](\w+|[^#]|$)|((?<=[\s\W])|^)[@]([a-zA-Z0-9_]+|$))/gm,
-        '',
-      )
+      // .replace(
```

Comment:
Let's remove if not needed

Reply:
I have a ticket to sort this out properly tomorrow, so we'll remove then if it's not needed 👍